package.json

```
@@ -79,6 +79,10 @@
         "metals.scalafmtConfigPath": {
           "type": "string",
           "markdownDescription": "Optional custom path to the .scalafmt.conf file.\n\nShould be relative to the workspace root directory and use forward slashes `/` for file separators (even on Windows)."
+        },
+        "metals.customRepositories": {
+          "type": "string",
```

I would probably make this 

```suggestion
          "type": "array",
          "items": {
             "type": "string"
          },
```

So that we don't have to arbitrarily pick a separator to parse the repositories (space, comma, semicolon?).

This has the downside of making the setting only editable from the `settings.json` file, since configurations of type `array` aren't yet supported the UI (see https://github.com/microsoft/vscode/issues/62440)

---

Indeed.